dev/Repeater/APITests/InspectingDataSourceTests.cs

```diff
@@ -402,6 +451,90 @@ public WinRTVectorChangedEventArgs(CollectionChange change, int index)
             }
         }
 
+        public class ObservableDataSource<T> : IReadOnlyList<T>, INotifyCollectionChanged, IKeyIndexMapping
```

Comment:
Can you add this in a separate file perhaps under the common folder ?

Reply:
Moved to a different file and updated the name of the collection to be more descriptive.